scipy/optimize/_differentialevolution.py

```diff
@@ -22,7 +22,7 @@
 def differential_evolution(func, bounds, args=(), strategy='best1bin',
                            maxiter=1000, popsize=15, tol=0.01,
                            mutation=(0.5, 1), recombination=0.7, seed=None,
-                           callback=None, disp=False, polish=True,
+                           callback=None, full_population=False, disp=False, polish=True,
```

Comment:
We would want to add the new parameter at the end to ensure backward compatibility.